pkg/operator/ceph/pool/controller.go

```diff
@@ -387,7 +388,8 @@ func createPool(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo,
 
 	logger.Infof("initializing pool %q", p.Name)
 	args := []string{"pool", "init", p.Name}
-	output, err := cephclient.NewRBDCommand(context, clusterInfo, args).Run()
+	cmd := cephclient.NewCephCommand(context, clusterInfo, args)
+	output, err := cmd.RunWithTimeout(exec.CephCommandsTimeout)
```

Comment:
It might be better to change the implementation of `NewRBDCommand()` to set a nonzero timeout. @travisn do you recall a reason why the RBD commands don't have a timeout by default?

Reply:
The [Run()](https://github.com/rook/rook/blob/6c50530737488c105e596c284a3c2fbe374d2460/pkg/daemon/ceph/client/command.go#L190-L192) method resets any timeout, so we would either need to remove that reset, or change all the callers to be `RunWithTimeout()`. I'd lean towards the latter change so we can be explicit about when the timeout is being used. Then we really get into the intent of #9557, so perhaps we should go ahead with this small change for now and revive #9557 for a bigger change.

Reply:
it's a `rbd` cmd how we can run it as a `ceph` cmd??

Reply:
@zhucan

Reply:
@zhucan, @parth-gr is right, it is `rbd` command and not `ceph` command. If you look at the code you will find `NewRBDCommand` uses `RBDTool` which `"rbd"` and `NewCephCommand` uses `CephTool` which is `"ceph"`

Also, we have a method `ExecuteRBDCommandWithTimeout`  [here](https://github.com/rook/rook/blob/master/pkg/daemon/ceph/client/command.go#L220) maybe we can use this. 

@travisn @BlaineEXE ^^^

Reply:
@subhamkrai Agreed on that approach

In the operator log it shows that `ceph pool init` is being run instead of `rbd pool init`.

Reply:
@subhamkrai ok, thanks